ext/src/test/java/com/playone/mobile/common/NotValidValueTest.kt

```diff
@@ -12,7 +13,7 @@ class MockClass {
 }
 
 data class MockDataClass(var _abc: Int = INVALID_INT) {
-    var abc by NotValidVar(_abc)
+    var abc by NotValidVar(if (_abc != INVALID_INT) _abc else INVALID_INT)
```

Comment:
`_abc.takeIf{ INVALID_INT != it } ?: INVALID_INT`
will be better?!

Reply:
Or we might make a function for checking the invalid value.

`_abc.takeIf(Int::isValid) ?: INVALID_INT` or something.